### PR TITLE
wkt macro to create geo-types

### PR DESCRIPTION
**Pull request by michaelkirk**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

Inspired by #1061. I don't think it necessarily replaces #1061, but it might obviate it. It's arguable we might want both (or maybe neither 😢)... how do people feel?

I've applied the new macro to a sampling of tests to give people an idea of where it's nice / less nice.

### CODE REVIEW

File: geo-types/src/geometry/multi_point.rs [hunk 1]
```diff
@@ -235,53 +235,50 @@ mod test {
...
     #[test]
     fn test_abs_diff_eq() {
         let delta = 1e-6;
 
-        let multi = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10., y: 10.]]);
+        let multi = wkt! { MULTIPOINT(0. 0.,10. 10.) };
 
-        let multi_x = MultiPoint::new(vec![point![x: 0., y: 0.], point![x: 10.+delta, y: 10.]]);
+        let mut multi_x = multi.clone();
+        *multi_x.0[0].x_mut() += delta;
```
Comment (michaelkirk) [inline geo-types/src/geometry/multi_point.rs:267]: Here's an example of where it's less nice. Unlike the `serde_json::json!` macro, you can't embed expressions, the `wkt!` macro only accepts literals.

**Comment by culebron** [comment]
I like this, I think this is better that what I wrote.

**Comment by urschrei** [comment]
This is great!

File: geo-types/src/macros.rs [hunk 2]
```diff
@@ -262,15 +260,11 @@ macro_rules! polygon {
             $crate::line_string![
                 $($exterior_coord), *
             ],
-            <[_]>::into_vec(
```
Comment (michaelkirk) [inline geo-types/src/macros.rs:265]: With the nostd build, we need to give a fully qualified path to the vec macro. That's the primary change in this commit.

But also I noticed these into_vec calls. I don't think we were gaining anything by building an array and then boxing it into a Vec vs. building the `vec!` directly, so I've changed it.

**Comment by urschrei** [comment]
I'd like to merge this. Anyone else want to chime in? Speak now (or any time before tomorrow morning)

**Comment by urschrei** [comment]
now what

**Pull request merged**